Dockerfile-proxy

```diff
@@ -1,7 +1,7 @@
 ARG RUNTIME_IMAGE=debian:buster-20200514-slim
 
 FROM debian:buster-20200514-slim as fetch
-RUN apt-get update && apt-get install -y ca-certificates curl
+RUN apt-get update && apt-get install -y ca-certificates curl jq
```

Comment:
Is this change actually needed? Where does `jq` get invoked?

Reply:
It is called when we don't specify which version to retrieve.
Ref: https://github.com/linkerd/linkerd2/blob/main/bin/fetch-proxy#L16